m0kv_metadata_parsing_tool/m0kv_metadata_parsing_tool.sh

```diff
@@ -26,24 +26,12 @@ reset=$(tput sgr0)
 readonly probable_delete_index='0x7800000000000000:0x100003'
 readonly bucket_list_index='0x7800000000000000:0x100002'
 
-
 rm base64_encoder_decoder -f > /dev/null 2>&1
 rm /var/log/m0kv_metadata.log > /dev/null 2>&1
 make > /dev/null 2>&1
 
 ##### Finding path of m0kv in cortx-s3server #####
-cd ..
-for line in $(find -name m0kv) 
-do
-    temp=$(echo "$line" | cut -c3-)
-    val=$(more "$temp" | grep -a ' temporary wrapper script for .libs/m0kv' ) 
-    if [[ ! -z "$val" ]]
-    then
-        m0kv_PATH=$(echo "$temp")
-    fi
-done
-cd m0kv_metadata_parsing_tool
-
+m0kv_PATH=$(find / ! -path /proc -name m0kv -type f | head -n1)
```

Comment:
Can't we simply hard code the path, found on HW ?

Reply:
No, because tool is for both dev and production environment. So if hardcode path it can fail on dev environment.